orm/models.py

```diff
@@ -266,6 +266,14 @@ async def create(self, **kwargs):
         instance.pk = await self.database.execute(expr)
         return instance
 
+    async def paginate(self, page:int = 1, per_page: int = 10):
```

Comment:
It's fine we provide default values?
Also, what is the best approach if the user pass a zero or negative number to page and per_page args. Do some if/ else checks and raise a exception or delegate to the user the responsibility to ensure that values are non zero positive integers?